test.py

```diff
@@ -976,7 +970,10 @@ def __init__(self, test_no: int, shortname: str, casename: str, suite) -> None:
         self.xmlout = os.path.join(self.suite.options.tmpdir, self.mode, "xml", self.uname + ".xunit.xml")
         self.server_log: Optional[str] = None
         self.server_log_filename: Optional[pathlib.Path] = None
```

Comment:
These strings can be deleted since you are clearing them one more time in next two lines.

Reply:
Nice catch :) Fixed